### PR TITLE
operf-micro.0.2 and operf-macro.0.2

### PR DESCRIPTION
**Pull request by Michaaell**

### CODE REVIEW

File: packages/operf-micro/operf-micro.0.2/opam [hunk 1]
```diff
@@ -0,0 +1,12 @@
+opam-version: "1.2"
+maintainer: "Pierre Chambart <pierre.chambart@ocamlpro.com>"
+authors: "Pierre Chambart <pierre.chambart@ocamlpro.com>"
+homepage: "http://www.typerex.org/operf-micro.html"
+bug-reports: "http://github.com/OCamlPro/operf-micro/issues"
+license: "GPLv2"
```
Comment (yallop) [inline packages/operf-micro/operf-micro.0.2/opam]: The OPAM file says the license is GPLv2, but the repository [says](https://github.com/OCamlPro/operf-micro/blob/master/LICENSE) it's MIT.

**Comment by yallop** [comment]
@Michaaell: could you update the license field?

**Comment by yallop** [comment]
Thanks!

**Comment by avsm** [comment]
It looks like `operf-macro` is actually uninstallable:

```
The following dependencies couldn't be met:
  - operf-macro -> perf
Your request can't be satisfied:
  - No package named perf found.
```

**Comment by yallop** [comment]
Oops -- not sure how I missed that.   @Michaaell, how would you like to resolve this?  Perhaps the easiest thing is to add [perf](https://github.com/OCamlPro/ocaml-perf) to the repository, if it's ready for release, but we could back out `operf-macro` instead.

**Pull request merged**